jongo/1.2.wso2v1/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We usually user major.minor.patch pattern for the file name and version. file name should be like 1.2.0.wso2v1